scripts/dockerlib.sh

```diff
@@ -44,6 +45,9 @@ LOG_FILE=$LOG_DIR/docker-handler.log
 CONFIG_FILE=$(ls $CONFIG_DIR | grep -E ^[0-9]+.settings$ | sort -n | tail -n 1)
 STATUS_FILE=$(echo $CONFIG_FILE | sed s/settings/status/)
 
+CONFIG=$CONFIG_DIR/$CONFIG_FILE
```

Comment:
@gabrielhartmann I'm no bash scripting expert, maybe lowercase (the current state) was the correct convention for this. Not sure if uppercase/lowercase means anything in bash but this is not exactly like a variable for a magic string, it's just a variable, composed from other variables.